Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py

```diff
@@ -0,0 +1,192 @@
+import os
+import traceback
+
+import demistomock as demisto  # noqa: F401
+import requests
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable warning for insecure requests when cert validation is disabled
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Get dict of CT instance parameters and command args
+CT_PARAMS = demisto.params()
+CMD_ARGS = demisto.args()
+
+SCAN_OPTIONS = ['pr', 'nw', 'nc', 'st', 'sc', 'ru', 'co', 'lo', 'ns', 'wb', 'fs']
+CT_SERVER = CT_PARAMS['server']
+REST_PORT = CT_PARAMS['rest_port']
+API_KEY = CT_PARAMS['api_key']
+USER = CT_PARAMS['credentials']['identifier']
+PASSWORD = CT_PARAMS['credentials']['password']
+VERIFY_SERVER_CERT = False
+REQ_HEADERS = {'restApiKey': API_KEY}
+USE_PROXY = CT_PARAMS['use_proxy']
+
+
+def IS_2XX(x):
+    return (x / 100) == 2  # Returns true if status code (int) is 2xx
+
+
+# Delete request proxy environment variables if user does not want to use proxy
+if not USE_PROXY:
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+# Dict of Cyber Triage REST APIs currently used
+REST_APIS = {
+    'check_creds': f'https://{CT_SERVER}:{REST_PORT}/api/correlation/checkcredentials',
+    'start_collection': f'https://{CT_SERVER}:{REST_PORT}/api/livesessions'
+}
+
+
+def make_rest_call(rest_api, method, json_data=None, headers=REQ_HEADERS, verify_cert=VERIFY_SERVER_CERT):
+    """Method to make rest calls (using requests) and handle error scenarios.
+
+    Args:
+        rest_api (str): a string that represents a url to a rest api
+        method (str): a string that represents an http methods supported by requests (get, put, post, ...)
+        json_data (dict, optional): same as requests json arg. Defaults to None.
+        headers (dict, optional): same as requests headers arg. Defaults to REQ_HEADERS.
+        verify_cert (bool, optional): same as requests verify arg. Defaults to VERIFY_SERVER_CERT.
+
+    Returns:
+        dict: Dictionary with response, status_code, and exception keys. status_code will be -1 or status code
+              returned in response object. Exception will be empty unless an exception occurred. Response will
+              contain the json response from CT if a 2xx response occurred, otherwise it will represent an error
+              message string.
+    """
+    ret_msg = ''
+    http_status_code = -1
+    e = None
+
+    # Get the request function associate with the type of http request we want to make (ex. GET or POST)
+    try:
+        request_func = getattr(requests, method)
+    except AttributeError:
+        ret_msg = f'Invalid requests method: {method}'
+        return_error(ret_msg)
+    else:
```

Comment:
There's no `finally` in this `try-except`, so why do we need `else`?

Reply:
`finally` is intended primarily for cleanup whereas the `else` clause of a `try-except` block is logic flow where that is what happens if an error _doesn't_ occur.